test/test_datasets.py

```diff
@@ -1494,6 +1494,41 @@ def test_num_examples_test50k(self):
             assert len(dataset) == info["num_examples"] - 10000
 
 
+class MovingMNISTTestCase(datasets_utils.DatasetTestCase):
+    DATASET_CLASS = datasets.MovingMNIST
+    FEATURE_TYPES = (torch.Tensor,)
+
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(split=(None, "train", "test"), split_ratio=(10,))
+
+    def inject_fake_data(self, tmpdir, config):
+        raw_dir = os.path.join(tmpdir, self.DATASET_CLASS.__name__)
+        os.makedirs(raw_dir, exist_ok=True)
+        self._create_npy_file(raw_dir, "mnist_test_seq.npy")
+        return self._num_batch_size()
+
+    def _create_npy_file(self, root: str, filename: str) -> None:
+        batch_size = self._num_batch_size()
+        with open(os.path.join(root, filename), "wb") as fh:
+            np.save(
+                fh, np.random.rand(20, batch_size, 64, 64)
+            )  # The shape is (sequece lenth, batch_size, height, widht)
+
+    def _num_batch_size(self):
+        return 100
+
+    @datasets_utils.test_all_configs
+    def test_num_examples(self, config):
```

Comment:
This test should test the `len(dataset)` and that should always be the test equivalent of `10_000`, right? Why is there a difference between the different splits? What you are testing below should be in a separate `test_num_frames`, no?

Reply:
I misunderstood of `split_ratio`.